events/wc_druidic_events.txt

```diff
@@ -24,12 +26,27 @@ character_event = {
 		fury_of_wolf_human_event_trigger = yes
 		
 		NOT = { has_global_flag = ongoing_fury_of_wolf_chain_flag }
-		
-		is_primary_war_defender = yes
+		OR = { 
+			any_courtier_or_vassal = { has_artifact = book_of_ur }
+			any_courtier_or_vassal = { has_artifact = scythe_of_elune }
```

Comment:
I recommend you to use only one any_courtier_or_vassal and put `OR` operator that checks artifacts. any_ scopes are CPU heavy.